web/packages/design/src/Tooltip/HoverTooltip.tsx

```diff
@@ -222,3 +249,27 @@ const StyledContent = styled(Text)`
   max-width: 350px;
   word-wrap: break-word;
 `;
+
+/**
+ * Combines multiple refs into one that can be passed to a component.
+ *
+ * https://github.com/gregberge/react-merge-refs/tree/v2.1.1
+ * @example
+ * const Example = React.forwardRef(function Example(props, ref) {
+ *   const localRef = React.useRef();
+ *   return <div ref={mergeRefs([localRef, ref])} />;
+ * });
+ */
+function mergeRefs<T = any>(
```

Comment:
It's already implemented: https://github.com/gravitational/teleport/blob/f264844f7c8393389e744302f01776d86ee729d7/web/packages/shared/libs/mergeRefs.ts#L33-L45
If the problem is that it can't be accessed from the design package, then let's just move it here instead of duplicating.

Reply:
I must admit I was a bit lazy here :) Moved to `design`.